kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -71,6 +71,7 @@ public class NMDbusConnector {
     private static final String NM_DEVICE_PROPERTY_DEVICETYPE = "DeviceType";
     private static final String NM_DEVICE_PROPERTY_STATE = "State";
     private static final String NM_DEVICE_PROPERTY_IP4CONFIG = "Ip4Config";
+    private static final String NM_DEVICE_PROPERTY_CONFIGURATION = "connection";
```

Comment:
This is not a device property and is not called "configuration". This doesn't follow the already established naming convention.

Reply:
I think this got changed during the cherry picking mess. 

Is ..._connection okay?